script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -1,53 +1,53 @@
 <Page1>
-It has to be sliding.
+It has to be gliding.//Sliding -> falling makes no sense, but gliding -> falling does 滑走だ。
 No, the correct term is closer to falling.
-...But can I do it? It's up to fate whether I can make it there or not.
+...But can I do it? It's up to fate whether I'll make it there or not.
 <Page2>
-There's no way that it will end up being safe. Even if everything goes well, the impact of the landing will tear my muscles and break my bones.
+It's not safe by any stretch of the imagination. Even if everything goes well, the impact of the landing will tear my muscles and break my bones.
 Will I really be able to endure that pain?
 <Page3>
-I stop the projectiles in their tracks as manifestations of my worries.
+I stop the projectiles just as they are on the brink of poking my worries directly.//I think the idea here is that they're about to enter his grey matter and poke his thoughts? 俺の 迷いを 突くかのような 弾丸を 寸前で 止める。
 That was close. Thank goodness.
-Because of that, I can finally make my decision.
-I just need to bear the burden. From the beginning, I have always been insensitive when it comes to death.
-I, too, will kill without a second thought.
+With that, my mind is made up.//Thanks to that feels a bit J->E? Also duplicative of above おかげでこっちも覚悟が決まった。
+I just need to bear the burden. From the beginning, I have always been insensitive to the pain of death.//もとより、死の痛みには鈍い性質だ。
+Just as he does, I, too, will kill without a second thought.//ヤツがそうであるように、俺も、一秒<違|たが>わず“殺す”事だけを考察してやる。
 <Page4>
-I turn around and run to Arcueid in the rear.
-My energy is completely depleted from the excessive movement.
+I turn around and run back towards Arcueid.
+With all this excess movement, my energy is completely drained.//その余分な移動で、体力は完全に尽きた。
 I can't take even one more step forward.
-The enemy I must approach is 50 meters away on the other side.
+The enemy I must approach is a distant 50 meters away.//Over yonder for 彼方 feels a bit out of character 近づくべき敵は50メートル先の彼方だ。
 <Page5>
 It doesn't matter.
 In any case, the horizontal distance has no bearing on my choice.//ruby dots <横|・><軸|・><の|・><距|・><離|・>
 <Page6>
 "Arcueid, can you do that thing again from earlier?!"
 "What do you mean by 'that thing'?!"
 <Page7>
-I tell Arcueid about my intent.
+I tell Arcueid what I intend to do.//アルクェイドにこちらの意図を伝える。
 The explanation ends with a single word.
-After a moment of bewilderment, Arcueid looks at me and agrees as if she has no choice.
+After a moment of bewilderment, Arcueid looks at my face and agrees as if she has no choice.
 <Page8>
-"...If we do it on that scale, I'll run out of energy to spare.
-But you're okay with that, right? You won't be able to escape anymore."
+"...If we do it on that scale, my reserve power will be completely drained.//「……その規模でやるとわたしの余力もなくなるわ。
```

Comment:
I'll have no energy to spare

Reply:
I guess my nitpick with the original is that running out of energy to spare suggests you still have non-spare energy left, whereas I think this is more like a "this is it, this is all the energy, we're boned if it doesn't work" intent? Does this alternative appeal?
> If we do it on that scale, it will consume the last of my remaining strength.